README.md

```diff
@@ -82,6 +82,10 @@ Federated learning research is booming. We organize the papers by their targetin
 * FedLearner [[Github]](https://github.com/bytedance/fedlearner) ByteDance
 * Baidu PaddleFL [[Github]](https://github.com/PaddlePaddle/PaddleFL)
 * Nvidia Clara SDK [[Web]](https://developer.nvidia.com/clara)
+* [Flower.dev](https://flower.dev/)
+* [PaddleFL](https://github.com/PaddlePaddle/PaddleFL)
+* [LEAF](https://leaf.cmu.edu/)
+* [FATE](https://fate.fedai.org/)
```

Comment:
PaddleFL, FATE, and LEAF are included already. I think LEAF is more like a benchmark instead of a framework. Please remove the last three and keep Flower. Thank you.